hbase-hadoop-compat/src/main/java/org/apache/hadoop/hbase/regionserver/MetricsRegionServerSource.java

```diff
@@ -238,6 +238,10 @@ public interface MetricsRegionServerSource extends BaseSource, JvmPauseMonitorSo
   String MAX_COMPACTED_STORE_FILE_REF_COUNT = "maxCompactedStoreFileRefCount";
   String MEMSTORE_SIZE = "memStoreSize";
   String MEMSTORE_SIZE_DESC = "Size of the memstore";
+  String MEMSTORE_ONHEAP_SIZE = "memStoreOnHeapSize";
```

Comment:
let's keep `memStoreHeapSize` as Duo suggested and it is also referenced as memStoreHeapSize at multiple places?